include/z64play.h

```diff
@@ -0,0 +1,197 @@
+#ifndef Z64PLAY_H
+#define Z64PLAY_H
+
+#include "ultra64.h"
+
+#include "z64actor.h"
+#include "z64animation.h"
+#include "z64bgcheck.h"
+#include "z64camera.h"
+#include "z64collision_check.h"
+#include "z64cutscene.h"
+#include "z64eff_footmark.h"
+#include "z64environment.h"
+#include "z64frameadvance.h"
+#include "z64game_over.h"
+#include "z64game.h"
+#include "z64interface.h"
+#include "z64item.h"
+#include "z64light.h"
+#include "z64math.h"
+#include "z64message.h"
+#include "z64pause_menu.h"
+#include "z64player.h"
+#include "z64prerender.h"
+#include "z64save.h"
+#include "z64scene.h"
+#include "z64skybox.h"
+#include "z64sound_source.h"
+#include "z64transition.h"
+#include "z64view.h"
+
+// TODO: Move to its own header?
+typedef struct SequenceContext {
+    /* 0x0 */ u8 seqId;
+    /* 0x1 */ u8 ambienceId;
+} SequenceContext; // size = 0x2
```

Comment:
Personally I say just remove this struct. It isn't actually needed (and currently isn't even used everywhere it could/should be if we were going to keep it).

Reply:
I kinda like those members being grouped into a struct, but as you say there's zero code proof that this struct actually exists besides a leftovers string on OoT. I guess it could go

Reply:
Ah I missed that. Should probably keep in then and eventually in its own header yeah.

Reply:
don't think it really needs it's own header

Reply:
I ended up renaming this struct to `SceneSequences` given Fig's suggestion on https://github.com/zeldaret/oot/pull/1966#discussion_r1651701491

Reply:
sorry I missed this conversation before starting the other one over in oot. What oot debug string is being referred to? 
I also kinda dont feel like it necessarily has to be a struct, but am also okay with the SceneSequences suggestion I made

Reply:
nevermind I found it `BGM設定game_play->sound_info.BGM=[%d]`

Reply:
This printf on z_kankyo.c

```c

        PRINTF("\n\n\nBGM設定game_play->sound_info.BGM=[%d] old_bgm=[%d]\n\n", play->sceneSequences.seqId,
               ((void)0, gSaveContext.seqId));
```
https://github.com/zeldaret/oot/blob/2ce4742bf69faebb16ea1710c4d4e7902b1be0f1/src/code/z_kankyo.c#L2081

Reply:
right okay. well, `SceneSequences` or something like that is def an improvement over "sound info". if others are fine with it

Reply:
`SceneSequences` seems fine, and yeah after further thought it doesn't need its own header.

Reply:
Yeah, really it is something internal to play and how it is organizing its internal data extracted from the scene. So keeping it in play just makes sense